console/README.md

```diff
@@ -20,7 +20,14 @@ Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.
 
 ## Running end-to-end tests
 
-Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+```bash
+# Usage:
+./cypress.sh <action> <environment>
+
+# Examples:
+./cypress.sh open ./local.env
+./cypress.sh run ./dev.env
+```
```

Comment:
@eliobischof can you please describe the prerequisites. Please describe what the command should do.

It would also help if you elaborate wether the user only needs to start this commands, or if somebody needs to start the backend first 🤷

Reply:
Furthermore, `open` does open `cypress`, but should something happen then?

<img width="1432" alt="Screenshot 2022-01-11 at 21 04 34" src="https://user-images.githubusercontent.com/9879976/149013326-99b4e39a-08ef-4765-9ca4-ac09809d1f7f.png">

Reply:
If I `run` it (without starting a backend of any form). I get this error.

<img width="1031" alt="Screenshot 2022-01-11 at 21 07 20" src="https://user-images.githubusercontent.com/9879976/149013667-e86a6edc-1ab6-4374-a0bd-fd1bb2ea16f7.png">

Reply:
> @eliobischof can you please describe the prerequisites. Please describe what the command should do.
> 
> It would also help if you elaborate wether the user only needs to start this commands, or if somebody needs to start the backend first shrug

I've linked to the quickstart guide where the e2e testing process is described better. Also, I've enriched the quickstart guide by node and npm requirements. As the dev section is a bit longer now, we could consider extracting it into another file now?

Reply:
Ok I will check this again later today